src/options/options.html

```diff
@@ -1,104 +1,68 @@
-            </div>
-            <input
-              type="text"
-              name="token"
-              x-model="token"
-              autocomplete="off"
-            />
+            " style="height: 30px; max-width: inherit;" />
+      </div>
+      <div class="mx-auto"></div>
+      <div class="justify-end mx-2">
+        <a class="no-underline" href="https://app.orbit.love" target="_blank" rel="noopener">Open Orbit app</a>
+      </div>
+    </header>
+    <div class="pt-6 mx-auto w-full max-w-screen-sm">
+      <h1 class="mb-6">Orbit Extension Settings</h1>
+      <div x-data="orbit()">
+        <div class="field">
+          <div class="mb-2 text-lg">
+            <label for="token">API Token</label>
           </div>
+          <input type="text" class="form-input py-2 px-3 bg-gray-100 rounded border-none" name="token" x-model="token"
+            autocomplete="off" />
+        </div>
 
-          <button id="save" @click="fetchWorkspaces()" class="mr-2">
-            Check token
-          </button>
-          <span
-            x-show="tokenCheckStatus.success !== undefined"
-            x-text="tokenCheckStatus.message"
-            x-bind:class="tokenCheckStatus.success ? 'text-green-500' : 'text-red-500'"
-          ></span>
+        <button id="save" @click="fetchWorkspaces()" class="mr-2">
+          Check token
+        </button>
+        <span x-show="tokenCheckStatus.success !== undefined" x-text="tokenCheckStatus.message"
+          x-bind:class="tokenCheckStatus.success ? 'text-green-500' : 'text-red-500'"></span>
 
-          <div
-            x-show="tokenCheckStatus.success && workspaces && workspaces.length > 0"
-          >
-            <div class="mt-2">
-              <div class="field">
-                <div class="text-lg mb-2">
-                  <span class="label-like">Select a workspace</span>
-                </div>
-                <template
-                  x-if="workspaces"
-                  x-for="workspace in workspaces"
-                  :key="workspace"
-                >
-                  <label class="inline-flex items-center mr-2">
-                    <input
-                      type="radio"
-                      class="form-radio"
-                      name="workspace"
-                      x-bind:value="workspace.attributes.slug"
-                      x-model="selectedWorkspaceSlug"
-                    />
-                    <span
-                      class="ml-2"
-                      x-text="workspace.attributes.name"
-                    ></span>
-                  </label>
-                </template>
+        <div x-show="tokenCheckStatus.success && workspaces && workspaces.length > 0">
+          <div class="mt-2">
+            <div class="field">
+              <div class="mb-2 text-lg">
+                <span class="label-like">Select a workspace</span>
               </div>
+              <template x-for="workspace in workspaces" :key="workspace.id">
```

Comment:
Notable difference to draw attention to - removed the `x-if="workspaces"` check here. This check is already made on line  41 (`<div x-show="tokenCheckStatus.success && workspaces && workspaces.length > 0">`) & the repeat was causing this to break by rendering an additional radio button with no associated workspace